src/patterns/question-pages/progress/index.njk

```diff
@@ -4,7 +4,7 @@ layout: layout-example.njk
 ignore_in_sitemap: true
 ---
 
+<span class="govuk-caption-xl">Question 3 of 9</span>
```

Comment:
Not blocking, this information seems like it'd be useful.

Reply:
in what sense?

Reply:
I just wondered if knowing which question you're on is something useful when scanning headings.